spec/1_base.adoc

```diff
@@ -6,7 +6,9 @@ All specified table, view, column, trigger, and constraint name values SHALL be
 
 [NOTE]
 ====
-For maximum interoperability, all GeoPackage table, view, column, trigger, and constraint name values SHOULD start with a lowercase character and only include lowercase characters, numbers 0-9, and underscores (`_`).
+For maximum interoperability, all GeoPackage table, view, column, trigger, and constraint name values SHOULD start with a lowercase character and only include lowercase characters, numbers 0-9, underscores (`_`), and not exceed 128 characters in length.
```

Comment:
better now?

Reply:
Yes, that's the point, uniqueness must be ensured in the first bytes. Long identifiers are an interoperability risk with older but not yet really legacy systems. Hopefully, this issue will be less and less relevant in the future (I pray for it!). 

As Spaniard there is an additional consideration. For example, the following SQL creates a valid table in sqlite:
```sql
CREATE TABLE españa(aragón int);   
```
I wonder if this may cause interoperability problems in some scenarios. Should `character` be interpreted only as `a-z` or should it also include letters with diacritical marks and non-Latin letters? Whatever the case, it must be made explicit. Probably the best is to restrict lowercase characters to `a-z` because the table `gpkg_contents` provides human readable Unicode names.